.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "deps/fmtlib"]
 	path = deps/fmtlib
 	url = https://github.com/fmtlib/fmt.git
+[submodule "deps/imgui"]
+	path = deps/imgui
+	url = https://github.com/ocornut/imgui.git
```

Comment:
I don't think it's a good idea to have a hard dependency on this via a submodule. Especially given that it's for an optional debugging tool. It would be much better off installed in the system like Boost.

Reply:
I'm generally not yet entirely convinced that we should merge this into the Solidity repo - and am rather wondering, if we should make it a repo in https://github.com/ethdebug pulling in solidity as submodule instead.

Reply:
yeah - for me it would be fine.. I just need something like that for implementing upcoming ethdebug features.. where it would exactly live is not really important for me.. I guess https://github.com/ethdebug might be the best place for it

Reply:
Yeah, it's definitely useful - and did spark quite a bit of progress - and prevented us from an annoying issue in the release in the end :-). But yeah, I'm still skeptical about having GUI tools in the main repo in general (together with all the infrastructure like that submodule and all that) - so to me a repo in https://github.com/ethdebug seems perfect, if that works nicely!

Reply:
yeah.. maybe the best is if we have this at first e.g. just under my github somewhere until it becomes more useful.. for me it would have been ok to have it here - because it would have been somehow hidden.. I agree that  https://github.com/ethdebug would be the perfect place, but as long it is not supporting some cool things, its probably better not to present this prominantly within the ethdebug repo directly